### PR TITLE
Moves authomaticallAdjustsScrolls to the base ARComponentVC 

### LINKED ISSUE #370: gene view header area is clashing with back button
on the left is our auction sale VC, on the right is current gene view

looks like the header area needs to increase in size to get the button to not clash with the fitered data

image

### PR DESCRIPTION
**Pull request by orta**
so it applies to all subclasses - should fix #370 and #368

### CODE REVIEW

File: Pod/Classes/ViewControllers/ARComponentViewController.m [hunk 1]
```diff
@@ -26,6 +26,7 @@ - (instancetype)initWithEmission:(AREmission *)emission
 - (void)viewDidLoad;
 {
   [super viewDidLoad];
+    self.automaticallyAdjustsScrollViewInsets = NO;
```
Comment (alloy) [inline Pod/Classes/ViewControllers/ARComponentViewController.m]: Why not in init? And has incorrect indentation 😱

Comment (orta) [inline Pod/Classes/ViewControllers/ARComponentViewController.m]: ideally shouldn't really be using properties in init, especially ones with side effects

did it in VS Code, will change

**Comment by orta** [comment]
👍

**Comment by alloy** [comment]
Grazie 👍

**Pull request merged**